cluster/gce/gci/configure-helper.sh

```diff
@@ -758,9 +758,55 @@ contexts:
 EOF
   fi
 
-if [[ -n "${GCP_IMAGE_VERIFICATION_URL:-}" ]]; then
-    # This is the config file for the image review webhook.
-    cat <<EOF >/etc/gcp_image_review.config
+  if [[ ${WEBHOOK_GKE_EXEC_AUTH:-} ]]; then
```

Comment:
Wrap parameter expansions in quotes here and below. Be explicit about what the condition is checking  a -n.

Reply:
Done.